Fitness/Models/PersonalLink.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  File.swift
+//  Fitness
+//
+//  Created by Cameron Hamidi on 3/21/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import Foundation
+
+struct PersonalLinkObject {
```

Comment:
Can we make this name PersonalLink and move the enum declaration inside the struct and call that LinkType or Site or something?

Reply:
Will do